FastSimulation/Tracking/plugins/SimTrackIdProducer.cc

```diff
@@ -30,11 +30,29 @@
 SimTrackIdProducer::SimTrackIdProducer(const edm::ParameterSet& conf)
 {
   //Main products
-  produces<std::vector<int> >(); 
+  produces<std::vector<unsigned int> >();
 
   // Input Tag
-  edm::InputTag trackCollectionTag = conf.getParameter<edm::InputTag>("trackCollection"); 
+  edm::InputTag trackCollectionTag = conf.getParameter<edm::InputTag>("trajectories"); 
```

Comment:
why trajectories? it is clearly a 'trackCollection' and not a trajectoryCollection!